deps/simdutf/README.md

```diff
@@ -0,0 +1,14 @@
+# simdutf
+
+This project boosts unicode validation and transcoding performance by 
+utilizing SIMD operations where possible.
+
+The source is pulled from: https://github.com/simdutf/simdutf
+
+Active development occurs in the default branch (currently named `master`).
+
+## Updating
+
+```sh
+$ git clone https://github.com/simdutf/simdutf
+```
```

Comment:
See [tools/dep_updaters/README.md#simdutf](../../tools/dep_updaters/README.md#simdutf) for instructions.